golang/godep.md

```diff
@@ -110,6 +112,55 @@ godep: no packages can be updated
 $ godep update github.com/Clever/worker/...
 ```
 
+## How do I do X?
+
+### first vendor save
```

Comment:
nit: is there a better way to phrase this? "How do I do first vendor save" feels off